src/Tribe/Tickets_Handler.php

```diff
@@ -1476,9 +1507,10 @@ class="tribe-ticket-field-order"
 			<td class="ticket_edit">
 				<?php
 				printf(
-					"<button data-provider='%s' data-ticket-id='%s' class='ticket_edit_button'><span class='ticket_edit_text'>%s</span></a>",
+					"<button data-provider='%s' data-ticket-id='%s' title='%s' class='ticket_edit_button'><span class='ticket_edit_text'>%s</span></a>",
 					esc_attr( $ticket->provider_class ),
 					esc_attr( $ticket->ID ),
+					sprintf( esc_attr__( '( Ticket ID: %d )', 'tribe-tickets' ), esc_attr( $ticket->ID ) ),
```

Comment:
I suppose it might just be better to wrap the whole sprintf() in an esc_attr() and remove the two additional esc_attr calls that have to happen here. Still fine with this either way.